test/inductor/test_aot_inductor.py

```diff
@@ -4203,5 +4204,5 @@ class AOTInductorTestABICompatibleCuda(AOTITestCase):
     from torch._inductor.test_case import run_tests
 
     # cpp_extension N/A in fbcode
-    if HAS_CUDA or sys.platform == "darwin" or HAS_CPU:
+    if HAS_CUDA or sys.platform == "darwin" or (not HAS_CUDA and HAS_CPU):
```

Comment:
This change seems logically equivalent to the original one.

Reply:
There is a little difference between `HAS_CPU` and `(not HAS_CUDA and HAS_CPU)`. `HAS_CUDA` will check if cuda is available and cuda device major version >=7. In the previous check, cuda with major version 6 will be included in the test, because it has CPU. So I use `(not HAS_CUDA and HAS_CPU)` to filter out the above scenario, and make sure the test can cover CPU only cases.